cw/capture_ecdsa_cw310.yaml

```diff
@@ -1,8 +1,15 @@
 device:
   fpga_bitstream: objs/lowrisc_systems_chip_earlgrey_cw310_0.1_ecdsa.bit
   fw_bin: objs/ecc_serial_fpga_cw310.bin
-  # Target operating frequency
-  pll_frequency: 22000000
+  # Target operating frequency: 
+  # for ecdsa-simple: 25-50MHz (alignment issues above 66MHZ)
+  # for ecdsa-stream: 15-20MHz (sampling_rate-adc_resolution tradeoff)
+  # More details on Husky stream mode can be found in the following links:
+  # https://github.com/lowRISC/ot-sca/issues/106#issuecomment-1359530023
+  # https://rtfm.newae.com/Capture/ChipWhisperer-Husky/#streaming-mode
+  # A diagram showing the clocking structure for optentitan CW310:
+  # http://drawings/1oVABuYbeXq5lwneQMZ49D5MBFfBJoSxz6V1A17j4XDo
```

Comment:
This link doesn't work. What you can do is include it in this PR as .png and then link to that. On the other hand, you don't really refer to this in the file or explain what is meant. So I think it's fine to skip this link in the source file. But it would be worth to add it to the PR for discussion.

Reply:
I removed the link from the source code. Added it to PR [(doc/img)](https://github.com/lowRISC/ot-sca/pull/109/files#diff-f0daec42d8aa0207b9e8566bc603bf2f0d3467b5a1a4a52c67184ab75080f9c5) for discussion and future reference.